plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -465,6 +485,75 @@ public void testDeletingEntireFileWithNonTupleDomainConstraint()
         assertThat(this.loadTable(tableName).newScan().planFiles()).hasSize(1);
     }
 
+    @Test
+    public void testMultiplePartitionedWithAggregationPushdown()
+    {
+        Session clientSession = sessionWithAggregationPushdown();
+
+        String tableName = format("test_%s_with_aggregation_pushdown", "multiple_partition");
+        assertUpdate(format("CREATE TABLE %s (userid int, country varchar, event_date date, state varchar) %s",
+                tableName, "WITH (partitioning = ARRAY['event_date', 'country'])"));
+
+        assertUpdate(format("INSERT INTO %s VALUES (1, 'USA', DATE '2022-11-01', 'California'), (2, 'USA', DATE '2022-11-01', 'Ohio')", tableName), 2);
+        assertUpdate(format("INSERT INTO %s VALUES (3, 'IND', DATE '2022-11-01', 'Delhi'), (4, 'IND', DATE '2022-11-01', 'MP')", tableName), 2);
+        assertUpdate(format("INSERT INTO %s VALUES (5, 'FRA', DATE '2022-11-02', 'Brittany'), (6, 'USA', DATE '2022-11-02', 'Corsica')", tableName), 2);
+        assertUpdate(format("INSERT INTO %s VALUES (7, 'USA', DATE '2022-11-04', 'Nevada')", tableName), 1);
```

Comment:
Can you please express the test queries in terms of tpch schema?
`nation` or `region` are not smaller than this test table. This would allow reader to quickly know what are the expected test values, without learning yet another test table.

You can eg create a partitioned `nation` table like this

```
CREATE TABLE nation_p WITH (partitioning=ARRAY['regionkey'])
AS SELECT * FROM tpch.tiny.nation;
```